src/main/java/com/revature/controllers/AddressController.java

```diff
@@ -0,0 +1,66 @@
+package com.revature.controllers;
+
+import java.util.HashSet;
+import java.util.Optional;
+import java.util.Set;
+
+import javax.servlet.http.HttpSession;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.revature.dtos.AddressRequest;
+import com.revature.models.Address;
+import com.revature.models.User;
+import com.revature.services.AddressService;
+import com.revature.services.UserService;
+
+@RestController
+@RequestMapping("/api/addresses")
+@CrossOrigin(origins = {"http://localhost:4200", "http://localhost:3000"}, allowCredentials = "true")
```

Comment:
Change this to `@CrossOrigin(origins="*", allowedHeaders="*")` to allow frontend in prod environment to reach this controller without throwing a CORS error